### ISSUE TITLE
Accessibility: Elements must meet minimum color contrast levels

### ISSUE THREAD

**Issue by bedmison** [web-cat/web-cat-subsystem-Core#5]
Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds. Contrast ratios should be 4.5:1

This issue occurs in many places:
- #utils
- #roles 
- #mainMenu
- .title.module
- The term/course/assignment selectors
- The download link
- The regrade link
- Section titles
- Score summary boxes (green background with white text). Boxes with white text and green backgrounds appear in muliple places in WC. 
- Staff manual scoring link (both on the white and grey background)
- Submission number link (both on the white and grey background)
- Copyright statement
